packages/next/src/server/lib/router-utils/setup-dev.ts

```diff
@@ -437,6 +438,16 @@ async function startWatcher(opts: SetupOpts) {
       )
     }
 
+    async function loadFallbackBuildManifest(
+      pageName: string,
+      type: 'app' | 'pages' = 'pages'
+    ): Promise<void> {
+      fallbackBuildManifests.set(
+        pageName,
+        await loadPartialManifest(BUILD_MANIFEST, pageName, type)
+      )
+    }
```

Comment:
Could we just reuse the existing `loadBuildManifest` function and `buildManifests` map?

Reply:
I think you are correct

Reply:
changed